x-pack/metricbeat/module/mssql/performance/data_integration_test.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 func TestData(t *testing.T) {
-	t.Skip("Skipping `data.json` generation test")
+	//t.Skip("Skipping `data.json` generation test")
```

Comment:
Should be reverted?

Reply:
Yes, I think it's better just in case. AFAIK it only runs if it's launched with `-data` but it's better to skip it explicitly.